### PR TITLE
typos

### PR DESCRIPTION
**Pull request by igneus**
The a/an corrected according to https://english.stackexchange.com/questions/3368/a-an-preceding-a-parenthetical-statement

### CODE REVIEW

File: distribution/index.md [hunk 1]
```diff
@@ -761,7 +761,7 @@ Open `https://localhost/books/` in your browser:
 You can also choose to generate the code for a specific resource with the `--resource` argument (example:
 `pnpm create @api-platform/client --resource books`).
 
-The generated code contains a list (including pagination), a delete button, a creation and an edition form. It also includes
+The generated code contains a list (including pagination), a delete button, a creation and an editation form. It also includes
```
Comment (alexislefebvre) [inline distribution/index.md:764]: It looks like *editation* is not in any dictionary?

Comment (alexislefebvre) [inline distribution/index.md:764]: It looks like *editation* is not in any dictionary?

See https://en.wiktionary.org/w/index.php?go=Go&search=editation&title=Special%3ASearch&ns0=1 or https://www.merriam-webster.com/dictionary/editation

**Review by alexislefebvre — CHANGES_REQUESTED** [review]

File: distribution/index.md (hunk 1 above)
Comment (igneus) [reply distribution/index.md]: What a surprise.

"Edit form" seems to be the correct term - or at least a widely used one.

**Review by dunglas — APPROVED** [review]

**Comment by alanpoulain** [comment]
Thanks!

**Pull request merged**